Src/BootCamp.Chapter/Items/Gloves.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace BootCamp.Chapter.Items
+{
+    public class Gloves
+    {
+        private string _name;
```

Comment:
fields need to be exposed through propreties (exposed as public in the first place) in other for anything to consume them.

Reply:
Agreed - in their current state (all private) they may as well be redundant. I will implement the same for the rest of the items.